feat(helm chart): add unified helm installer for openebs local and replicated engines (#3704)
openebs/openebs: charts/templates/NOTES.txt

Successfully installed OpenEBS.

Check the status by running: kubectl get pods -n {{ .Release.Namespace }}

> The default values will install both Local and Replicated Engines. However,

Reviewer: ```suggestion
The default values install both Local and Replicated Engines. However,
```

Reviewer: I guess there's no way to "template" this doc depending on what the user installed?

Reviewer 2: Yes this doc this template friendly. What do you have in mind?

Reviewer: As discussed offline, this is fine for now.

> the Replicated Engine will require additional configuration to be fuctional.

Reviewer 2: ```suggestion
the Replicated Storage option will require additional configuration to be functional.
```

> The Local Engine offers non-replicated local storage using 3 different storage

Reviewer 2: ```suggestion
The Local options offer non-replicated local storage using 3 different storage
```

> backends, while the Replicated Engine provides replicated highly-available

Reviewer 2: ```suggestion
backends, while the Replicated one provides replicated highly-available
```

Author: Please re review the content, have tried to change is as per last naming discussion.
